terraform/gcp/security/gcp-compute-ip-forward.yaml

```diff
@@ -0,0 +1,22 @@
+rules:
+- id: gcp-compute-ip-forward
+  patterns:
+    - pattern: resource
+    - pattern-inside: |
+        resource "google_compute_instance" "..." {
+        ...
+        can_ip_forward = true
+        ...
+        }
+  message: >-
+    Ensure that IP forwarding is not enabled on Instances	

```

Comment:
What's the danger of IP forwarding? Does it allow for SSRF or something similar?

Reply:
When the IP Forwarding feature is enabled on a virtual machine's network interface (NIC), it allows the VM to act as a  router/packet forwarder and receive traffic addressed to other destinations.